### PR TITLE
Atom 8508 activate port api

### PR DESCRIPTION
**Pull request by babasahebk**
## Ticket
https://jira.bqsoft.com/browse/ATOM-8508

## Description
ITG Activate Port API
image
image
image

### CODE REVIEW

File: lib/conduit/plintron/views/layout.erb [hunk 1]
```diff
@@ -1,10 +1,10 @@
...
-<TRANSACTION_ID xsi:type="xsd:long"><%= transaction_id %></TRANSACTION_ID>
-<ENTITY xsi:type="xsd:string"><%= token %></ENTITY>
-<CHANNEL_REFERENCE xsi:type="xsd:string"><%= channel_reference %></CHANNEL_REFERENCE>
+<TRANSACTION_ID><%= transaction_id %></TRANSACTION_ID>
+<ENTITY><%= token %></ENTITY>
+<CHANNEL_REFERENCE><%= channel_reference %></CHANNEL_REFERENCE>
 <% if asynchronous %>
-<COMMUNICATION_METHOD xsi:type="xsd:int">1</COMMUNICATION_METHOD>
+<COMMUNICATION_METHOD>1</COMMUNICATION_METHOD>
 <% end %>
```
Comment (ufchrisg) [inline lib/conduit/plintron/views/layout.erb]: was there a Plintron reason we didn't just  include communication method and pass the value we want based on async or not? I feel like I remember there was, like Plintron had some weird "don't pass it if it's not async but if it's async pass it as a value of 1" but just need a refresher.

File: lib/conduit/plintron/parsers/activate_port.rb [hunk 2]
```diff
@@ -0,0 +1,8 @@
+require_relative "base"
+
+module Conduit::Driver::Plintron
+  class ActivatePort::Parser < Parser::Base
+    # Specify attributes here:
+    # attribute :attr_name
```
Comment (ufchrisg) [inline lib/conduit/plintron/parsers/activate_port.rb]: delete

File: lib/conduit/plintron/request_mocker/activate_port.rb [hunk 3]
```diff
@@ -0,0 +1,7 @@
+require_relative "base"
+
+module Conduit::Plintron::RequestMocker
+  # Request mocker for activate_port.
```
Comment (ufchrisg) [inline lib/conduit/plintron/request_mocker/activate_port.rb]: delete

File: lib/conduit/plintron/actions/activate_port.rb [hunk 4]
```diff
@@ -0,0 +1,10 @@
+require_relative "base"
+
+module Conduit::Driver::Plintron
+  class ActivatePort < Conduit::Driver::Plintron::Base
+    required_attributes [:imsi, :iccid], :mdn, :carrier_account, [:first_name, :last_name],
```
Comment (pedromartinez) [inline lib/conduit/plintron/actions/activate_port.rb]: First name or last name are required? Not both?

File: lib/conduit/plintron/views/layout.erb (hunk 1 above)
Comment (babasahebk) [reply lib/conduit/plintron/views/layout.erb]: Plintron don't like empty tags I think

**Pull request merged**